backend/functions/tests/user.test.js

```diff
@@ -1,4 +1,5 @@
 const path = require("path");
+const crypto = require("crypto");
```

Comment:
Built-in Node.JS library
- Ref: https://nodejs.org/api/crypto.html

Reply:
I see. What is the difference from the way we were creating test users before?

Reply:
essentially instead of using a fixed id such as `test-id-2`, we generate a random string such as `76e19d50-1419-40be-adb1-1e17963671f8`

Reply:
I was just wondering if there is a benefit to using crypto to generate a random ID instead of the static user-id for testing purposes. Since it's not actually real users. I mean it's nice of course. but just in case I'm overlooking something.

Reply:
Well this is for testing purposes and since GitHub Actions was running tests concurrently on 3 node versions, once one finishes, it'll delete the data it created in cleanup. Since our tests looks at the `uid` field, the cleanup might delete a document the other tests were looking at, so this is a way to solve that issue.

Reply:
Oh, I see. I was indeed overlooking that LOL. thanks for clarifying!